src/System.Management.Automation/utils/Telemetry.cs

```diff
@@ -666,6 +676,7 @@ internal static void SendPSCoreStartupTelemetry(string mode)
             try
             {
                 s_telemetryClient.TrackEvent("ConsoleHostStartup", properties, null);
+                s_startupEventSent = true;
```

Comment:
Should we be concerned about thread safety here?

Reply:
Addressed